db/cache/charts.go

```diff
@@ -416,7 +435,11 @@ func (charts *ChartData) Lengthen() error {
 	}
 
 	windows := charts.Windows
-	shortest, err = ValidateLengths(windows.Time, windows.PowDiff, windows.TicketPrice, windows.StakeCount)
+	// expectedMissedSlice shows the full slice size for the missed votes slice if
+	// the stake validation height was zero.
+	expectedMissedSlice := append(make(ChartUints, windows.stakeValid), windows.MissedVotes...)
```

Comment:
Generating slice just to spoof validation is not great. The zero-prepending solution solves this too.